rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy

```diff
@@ -104,12 +105,25 @@ class ScheduledExecutionService implements ApplicationContextAware, Initializing
                 defaultValue null
                 renderingOption('booleanTrueDisplayValueClass', 'text-warning')
             }.build(),
+            PropertyBuilder.builder().with {
+                string 'activityDefault'
+                title 'Activity Default'
```

Comment:
I think the title should be more descriptive, like "Activity Page Default Time Period"